### PR TITLE
Moving sqli arm templates to workbooks

### PR DESCRIPTION
**Pull request by leelabysani**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

**Comment by dougbrad** [comment]
I assume workbook changes are needed to actually use the templates?

**Comment by leelabysani** [comment]
> I assume workbook changes are needed to actually use the templates?

Yes, I thought it has to be done in two different PRs for the files to be available to make change in workbooks. May be I am wrong, I will check and update workbook files too

File: Workbooks/Workloads-AMA/Add monitoring virtual machine/Add monitoring virtual machine.workbook [hunk 1]
```diff
@@ -469,7 +469,7 @@
                     "componentIdSource": "parameter",
                     "componentId": "ResourceGroup",
                     "templateUriSource": "static",
-                    "templateUri": "https://azmonsolutions.blob.core.windows.net/workloadinsights/add-monitoring-vm-azuredeploy.json",
+                    "templateUri": "ARMTemplates/AddMonitoringVirtualMachine.armtemplate",
```
Comment (yves-chan) [inline Workbooks/Workloads-AMA/Add monitoring virtual machine/Add monitoring virtual machine.workbook:472]: This should read something like 'Community-Workbooks/Workloads-AMA/Add monitoring virtual machine/AddMonitoringVirtualMachine.armtemplate'

Pretty much the path of the github repo, except the physical folder path 'workbooks' is 'Community-Workbooks'

**Review by yves-chan — COMMENTED** [review]
Can you also check in: Workbooks/Workloads-AMA/Custom Telegraf/ and replace all the old url references also? They should be using the same templates. Once that is done I think we can merge this

**Comment by leelabysani** [comment]
> Can you also check in: Workbooks/Workloads-AMA/Custom Telegraf/ and replace all the old url references also? They should be using the same templates. Once that is done I think we can merge this

Done

**Review by yves-chan — APPROVED** [review]

**Review by dougbrad — APPROVED** [review]

**Pull request merged**